### PR TITLE
Add Appuser filter data

### PR DESCRIPTION
**Pull request by lakshmiSatyanarayana**
Added users filter by Employee role service

### CODE REVIEW

File: app/controllers/HomeController.java [hunk 1]
```diff
@@ -37,5 +37,11 @@ public Result index() {
           mailerClient.send(email);*/
           return ok(index.render("Your new application is ready."));
     }
+	
+	public Result getUsers(){
+		List<AppUser> appUsers = new ArrayList<AppUser>();
+		appUsers.stream().filter(user -> user.role.equals("EMPLOYEE")).collect(Collectors.toList());
```
Comment (Lakshmi-v) [inline app/controllers/HomeController.java:43]: @lakshmiSatyanarayana  you are filtering appusers list which will be always a empty list forever as per the folw here. Verify and fix this accordingly

File: app/controllers/HomeController.java [hunk 2]
```diff
@@ -37,5 +37,11 @@ public Result index() {
...
+		return ok(index.render("Your new application is ready."));
```
Comment (Lakshmi-v) [inline app/controllers/HomeController.java:44]: @lakshmiSatyanarayana  returning to index page from get users is not as per the requirement. please see

File: app/controllers/HomeController.java (hunk 1 above)
Comment (lakshmiSatyanarayana) [reply app/controllers/HomeController.java:43]: @Lakshmi-v  this is a test . ignore for this PR

File: app/controllers/HomeController.java (hunk 2 above)
Comment (lakshmiSatyanarayana) [reply app/controllers/HomeController.java:44]: @Lakshmi-v  This is test. ignore this

**Review by Lakshmi-v — APPROVED** [review]

**Pull request merged**